sahha-android/src/main/java/sdk/sahha/android/domain/interaction/PermissionInteractionManager.kt

```diff
@@ -191,7 +240,8 @@ internal class PermissionInteractionManager @Inject constructor(
 
     internal fun processStatuses(
         nativeStatus: Enum<SahhaSensorStatus>,
-        healthConnectStatus: Enum<SahhaSensorStatus>
+        healthConnectStatus: Enum<SahhaSensorStatus>,
+        isBelowOrEqualToAndroid9: Boolean = isBelowOrAndroid9
```

Comment:
I needed this parameter for unit testing - as it has no knowledge of the Android build or versions